Services/WebServices/ECS/classes/class.ilECSObjectSettings.php

```diff
@@ -119,31 +125,11 @@ protected function isActive(): bool
      */
     public function addSettingsToForm(ilPropertyFormGUI $a_form, $a_type): bool
     {
-        $this->logger->debug('Show ecs settings.');
-        if (!$this->isActive()) {
-            $this->logger->debug('Object type is not active. => no settings.');
-            return false;
-        }
+        $export_manager = ilECSExportManager::getInstance();
+        $exportable_participants = (new ilECSParticipantSettingsRepository())->getExportableParticipants($a_type);
```

Comment:
This renames the used variables from exportManager to export_manager, which causes issues in the rest of this method. Same issue exists with exportableParticipants.